apps/backend/src/services/polling_service.py

```
@@ -68,14 +70,15 @@ async def stop_polling(self) -> None:
         self.is_running = False
 
-        # Cancel all polling tasks
-        for device_id, task in self.polling_tasks.items():
-            if not task.done():
-                task.cancel()
-                try:
-                    await task
-                except asyncio.CancelledError:
-                    pass
+        # Cancel all polling tasks for all devices
+        for device_id, tasks in self.polling_tasks.items():
```

Ruff (0.12.2)

Loop control variable `device_id` not used within loop body

Rename unused `device_id` to `_device_id`

(B007)